CMakeLists.txt

```diff
@@ -160,6 +176,10 @@ if(USE_POSIX_COMMIT_CHECKS)
   target_compile_definitions(snmalloc_lib INTERFACE -DUSE_POSIX_COMMIT_CHECKS)
 endif()
 
+if(LIBC_HAS_GETENTROPY)
+  target_compile_definitions(snmalloc_lib INTERFACE -DLIBC_HAS_GETENTROPY)
```

Comment:
Please prefix snmalloc macros with `SNMALLOC_`.  All of our macros appear in the global namespace for consumers and so shouldn't start with an identifier that isn't ours.